scripts/initiate_ritual.py

```diff
@@ -72,6 +75,14 @@
     help="The filepath of a file containing newline separated staking provider addresses.",
     type=click.File("r"),
 )
+@click.option(
+    "--autosign",
+    is_flag=True,
+)
+@click.option(
+    "--heartbeat",
+    is_flag=True,
+)
```

Comment:
```suggestion
```suggestion
@click.option(
    "--heartbeat",
    help="Run a heartbeat check, whereby rituals will be initiated so every node is involved.",
    is_flag=True,
)
```
)
```